src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -20,16 +23,21 @@
     public static final String DEFAULT_NAME = "Amy Bee";
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "amy@gmail.com";
+    public static final String DEFAULT_INTERVIEWSLOT = LocalDateTime.MAX.format(DateTimeFormatter
+            .ofPattern(InterviewSlot.FORMAT_DATETIME_INPUT));
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
     public static final String DEFAULT_JOBTITLE = "Intern";
 
+
```

Comment:
You can remove this line as well while you are reverting the changes to lchokhoe.png.